app/manifest.webapp

```diff
@@ -7,7 +7,7 @@
     "48": "/img/omrs-button.png"
   },
   "developer": {
-    "name": "wanjikum",
+    "name": "andela-pupendo,kingisaac95, andela-wanjikum, andela-jomadoye",
     "url": "https://github.com/wanjikum/openmrs-owa-addonmanager"
```

Comment:
Could we have the url set to the main add-on manager repository url

Reply:
Yes sure. That can be changed.